.github/workflows/dev_ci.yaml

```diff
@@ -18,9 +18,6 @@ jobs:
       - name: Install dependencies
         run: npm ci
 
-      - name: Generate contract types
-        run: npm run build-types
```

Comment:
This only existed because, initially, we were not pushing the contract types.

Since we already push them, this is not needed.